envs/prod/variables.auto.tfvars

```diff
@@ -17,13 +17,107 @@ project_id = "my-prod-project"
 pubsub_topic_name = "vertex-pipelines-trigger"
 
 cloud_schedulers_config = {
-  training = {
-    name        = "training-pipeline-trigger",
-    region      = "europe-west4", # Must be the same as the App Engine region
-    description = "Trigger my training pipeline in Vertex",
-    schedule    = "0 0 * * 0",
-    time_zone   = "UTC",
-    # Relative path to payload JSON file for this environment
-    payload_file = "../../pipelines/xgboost/training/payloads/prod.json",
-  },
+  # Uncomment and amend as required
```

Comment:
@browningjp-datatonic 
There must be a better approach than 100 lines of commented code. 
Would rather have open in cloud shell with tensorflow and xgboost as two example approaches

Reply:
Agree - but probably the refactor needed outside of the scope of this PR